### ISSUE TITLE
X2Paddle Roadmap and Feature Request

### ISSUE THREAD

**Issue by jiangjiajun** [PaddlePaddle/X2Paddle#601]
我们在这里罗列了X2Paddle待计划和已完成的事项，如若你有任何新的需求或建议反馈，可以在此issue下回复。

## X2Paddle v1.4 [2021.07&&08]
- [ ] 支持TFLite模型格式转换，支持70+ OP，TensorFlow Hub 20+模型
- [ ] 集成PaddleLite的Opt工具

## X2Paddle v1.3 [2021.07.12 Released](https://github.com/PaddlePaddle/X2Paddle/releases/tag/v1.3.0)
- [ ] 训练代码转换支持
- - [x] [CRAFT](https://github.com/clovaai/CRAFT-pytorch)
- - [ ] yolov5[暂不支持]
- - [ ] DiscoGAN
-  [x] 去除to_tensor
- [ ] 完善API文档
- [x] 支持mmdetection预测模型转换
- - [x] YOLO-V3
- - [x] FCOS
- - [x] RetinaNet
- - [x] SSD
- - [x] Faster R-CNN
- - [x] FSAF
- - [ ] Mask R-CNN
- - [ ] CornerNet
- [x] 增加mmdetection模型转换文档

## X2Paddle v1.2.3 [2021.06.11 Released](https://github.com/PaddlePaddle/X2Paddle/releases/tag/v1.2.3)
- [x] ONNX2Paddle新增模型转换：[RetinaNet](https://github.com/open-mmlab/mmdetection/blob/master/configs/retinanet/retinanet_r50_fpn_1x_coco.py)、[FSAF](https://github.com/open-mmlab/mmdetection/blob/master/configs/fsaf/fsaf_r50_fpn_1x_coco.py)、[SSD](https://github.com/open-mmlab/mmdetection/blob/master/configs/ssd/ssd300_coco.py)、[Faster R-CNN](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py)

## X2Paddle v1.2.2 [2021.06.04 Released](https://github.com/PaddlePaddle/X2Paddle/releases/tag/v1.2.2)
- [x] Caffe Pooling：支持kernel_size=0的情况
- [x] ONNX Resize：支持3d Tensor的转换
- [x] 支持PaddlePaddle 2.0.x 中的paddle type

## X2Paddle v1.2.1 [2021.05.26 Released](https://github.com/PaddlePaddle/X2Paddle/releases/tag/v1.2.1)
- [x] 文档、代码更新适配Paddle2.1.0
- [x] 去除静态图用法
## X2Paddle v1.2 [2021.05.15 Released](https://github.com/PaddlePaddle/X2Paddle/releases/tag/v1.2.0)
- [x] 支持PyTorch训练代码转为PaddlePaddle训练代码
- [x] 新增PyTorch-PaddlePaddle对比文档
